colorama/ansitowin32.py

```diff
@@ -21,6 +21,23 @@ def is_a_tty(stream):
     return hasattr(stream, 'isatty') and stream.isatty()
 
 
+def check_windows_ansi_support():
+    try:
+        import winreg
```

Comment:
The module is named `_winreg` on Python 2.7 (notice the leading underscore). https://docs.python.org/2/library/_winreg.html